module-3/backend/week_one.md

```diff
@@ -5,7 +5,16 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What is `json` and why is it important?
+  * JSON stands for Java Script Object notation, which is a data format, easy to read and write for us and machines. It's important because it bridges that gap between humans and machine in how to transpose data.
```

Comment:
Check to make sure you've pulled from upstream master before you begin these. We had some additional questions to keep your M1, and M2 knowledge fresh.